src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```diff
@@ -395,18 +402,23 @@ protected List<List<String>> extractFields(final VariantContext vc) {
 
     private void addGenotypeFieldsToRecords(final VariantContext vc, final List<List<String>> records, final boolean errorIfMissingData) {
         for ( final String sample : samples ) {
+            final Genotype genotype = vc.getGenotype(sample);
             for ( final String gf : genotypeFieldsToTake ) {
-                if ( vc.hasGenotype(sample) && vc.getGenotype(sample).hasAnyAttribute(gf) ) {
+                if ( vc.hasGenotype(sample) && genotype.hasAnyAttribute(gf) ) {
```

Comment:
Pulling the genotype into a variable above before doing the `hasGenotype` check seems strange to me.

Reply:
that's.... a good point...